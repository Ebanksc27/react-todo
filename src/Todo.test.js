import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

test('renders Todo component without crashing', () => {
  // Provide mock props
  const removeTodo = jest.fn();
  render(<Todo id="1" task="Test Todo" removeTodo={removeTodo} />);
});

test('Todo component snapshot', () => {
  const removeTodo = jest.fn();
  const { asFragment } = render(<Todo id="1" task="Test Todo" removeTodo={removeTodo} />);
  expect(asFragment()).toMatchSnapshot();
});
