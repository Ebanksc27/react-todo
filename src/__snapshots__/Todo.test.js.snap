// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo component snapshot 1`] = `
<DocumentFragment>
  <div>
    <span>
      Test Todo
    </span>
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
